build/rsync_website.sh

```diff
@@ -12,4 +12,4 @@
 #
 #############################################################################################################################################################################
 
-rsync -Pavqr --protocol=28 --delete-after target/website/* optaplanner@filemgmt.jboss.org:www_htdocs/optaplanner
\ No newline at end of file
+rsync -Pavqr --protocol=28 --delete-after optaplanner-website-assembly/target/website-complete/* optaplanner@filemgmt.jboss.org:www_htdocs/optaplanner
```

Comment:
@mbiarnes could you please confirm if this script is the only way how we update the website? In other words, do I have to touch anything else if I change the directory structure of this repository?